src/ui/pages/Settings/Preferences.tsx

```diff
@@ -0,0 +1,157 @@
+          <VStack gap={0}>
+            {Object.values(CURRENCIES).map(({ code, name, symbol }) => (
+              <label key={code}>
+                <input
+                  type="radio"
+                  name="currency"
+                  value={code}
+                  defaultChecked={code === currency}
+                  className={cn(helperStyles.visuallyHiddenInput, styles.input)}
+                />
+                <FrameListItem className={styles.listItem}>
+                  <HStack
+                    gap={8}
+                    alignItems="center"
+                    justifyContent="space-between"
+                  >
+                    <HStack gap={8} alignItems="center">
+                      <UIText
+                        kind="headline/h3"
+                        style={{ width: 24, textAlign: 'center' }}
+                      >
+                        {symbol}
+                      </UIText>
+                      <UIText kind="body/accent">{name}</UIText>
+                    </HStack>
+                    <CheckIcon
+                      className={styles.checkIcon}
+                      style={{
+                        color: 'var(--primary)',
+                        width: 24,
+                        height: 24,
+                      }}
+                    />
+                  </HStack>
+                </FrameListItem>
+              </label>
+            ))}
+          </VStack>
+        </Frame>
+        <Spacer height={16} />
+        <StickyBottomPanel containerStyle={{ marginTop: 'auto' }}>
+          <HStack
+            gap={16}
+            style={{ padding: 16, gridTemplateColumns: '1fr 1fr' }}
+          >
+            <Button kind="regular" type="button" onClick={goBack}>
+              Reset
+            </Button>
+            <Button kind="primary">Save</Button>
+          </HStack>
+        </StickyBottomPanel>
+      </form>
+    </PageColumn>
+  );
+}
+
+function PreferencesMain() {
+  const { currency } = useCurrency();
+  useBackgroundKind({ kind: 'white' });
+
+  return (
+    <PageColumn>
+      <NavigationTitle title="Preferences" />
+      <PageTop />
+      <Frame>
+        <VStack gap={0}>
+          <FrameListItemLink to="currency">
+            <AngleRightRow>
+              <HStack
+                gap={24}
+                alignItems="center"
+                justifyContent="space-between"
+              >
+                <UIText kind="body/accent">Currency</UIText>
+                <UIText kind="small/regular" color="var(--neutral-500)">
+                  {CURRENCIES[currency].code.toUpperCase()}
+                </UIText>
+              </HStack>
+            </AngleRightRow>
+          </FrameListItemLink>
+        </VStack>
+      </Frame>
+      <PageBottom />
+    </PageColumn>
+  );
+}
+
+export function PreferencesPage() {
+  return (
+    <Routes>
+      <Route
+        path="/"
+        element={
+          <ViewSuspense>
+            <PreferencesMain />
```

Comment:
Why do we need an additional screen? We can display "currency" option in Settings

Reply:
This is definitely need a discussion :))
I will refactor save/reset button for now